packages/gatsby/package.json

```diff
@@ -46,13 +46,13 @@
     "domready": "^1.0.8",
     "dotenv": "^4.0.0",
     "eslint": "^4.19.1",
-    "eslint-config-react-app": "^2.1.0",
+    "eslint-config-react-app": "3.0.0-next.66cc7a90",
```

Comment:
Is this change needed? What is different in `3.0.0` compared to `2.1.0`?

Reply:
Yeah, it’s necessary as 2.1.0 is made using eslint-plugin-jsx-a11y@5, while 3.0.0 is made using eslint-plugin-jsx-a11y@6

Reply:
2.1.0 refers a deprecated rule from eslint-plugin-jsx-a11y@5

Reply:
Makes sense, thank you.